### PR TITLE
Release v4.0.0

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java [hunk 1]
```diff
@@ -21,18 +21,15 @@ public final class MacheteFileUtils {
   private MacheteFileUtils() {}
 
   public static String getSampleMacheteFileContents() {
-    // We're deliberately using \n rather than `System.lineSeparator()` here
```
Comment (PawelLipski) [inline frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: Huh TBD if this doesn't break anything on Windows

Comment (mkondratek) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: image

Comment (mkondratek) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: I see no errors

Comment (PawelLipski) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: Whoops sorry if I wasn't clear... pls check Color Scheme and Code Style on Windows:

image

Comment (mkondratek) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: image

Comment (mkondratek) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: _brak_

Comment (mkondratek) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: image

Comment (mkondratek) [reply frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/MacheteFileUtils.java:24]: wokrs! sorry, first i checked 3.8.0

**Pull request merged**